templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+	<h1>{{ object|yesno:"Update, Create" }} Car</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input type="submit" value="Submit">
+  </form>
```

Comment:
You should use 2 whitespaces indentations in your .html files. This is a code style issue. Please adjust the indentation for better code readability.

Reply:
done